# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/plasma-login-manager/hotfix.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

src/daemon/SocketServer.cpp:53:11: error: invalid use of incomplete type ‘class QDebug’
   53 |     qDebug() << "Socket server starting...";

--- plasma-login-manager/src/daemon/SocketServer.cpp.vanilla	2026-02-17 19:37:20.591589726 +0100
+++ plasma-login-manager/src/daemon/SocketServer.cpp	2026-02-17 19:37:42.567061097 +0100
@@ -22,6 +22,7 @@
 #include "SocketWriter.h"
 #include "Utils.h"
 
+#include <QDebug>
 #include <QLocalServer>
 #include <QLocalSocket>
 
